### ISSUE TITLE
Groups of groups group_vars file not read by ansible-playbook

### ISSUE THREAD

**Issue by sbocahu** [ansible/ansible#874]
Ansible-playbook does not read /etc/ansible/group_vars/parent_group when using this inventory:

```
[billythekid] 
mysuperhost

[parent_group:children]
billythekid
```

I tried to track down the issue but don't udnerstand where the bug comes from.

groups is [billythekid]  in ansible/playbook/play.py line 229
however mysuperhost has parent_group in its group list at the end of the inventory parsing (printed them in ansible/inventory/ini.py line 141)

What did i miss ?

**Comment by mpdehaan** [comment]
I probably need to tweak the code to look for ancestor groups here.

**Comment by mpdehaan** [comment]
fixed in 6de8c27, thanks for the report!
